README.md

```diff
@@ -228,26 +224,37 @@ GTMSessionFetcher *fetcher = [fetcherService fetcherWithURL:userinfoEndpoint];
 
 ### Saving to the Keychain
 
-You can easily save `GTMAppAuthFetcherAuthorization` instances to the Keychain using
-the included `GTMAppAuthFetcherAuthorization+Keychain` category.
+You can easily save `GTMAuthSession` instances to the Keychain using the `GTMKeychainStore` class.
 
 ```objc
+// Create a GIDKeychainStore instance
+GIDKeychainStore *keychainStore =
+    [[GIDKeychainStore alloc] initWithItemName:kGTMAppAuthExampleAuthorizerKey];
+    
+NSError *error;
+
 // Save to Keychain
-[GTMAppAuthFetcherAuthorization saveAuthorization:_authorization
-                                toKeychainForName:kGTMAppAuthExampleAuthorizerKey];
+[keychainStore saveAuthSession:self.authSession error:&error];
```

Comment:
Worth adding a comment that says the `authSession` is saved with the key given during the `KeychainStore`'s initialization?

Reply:
Added an explanation in the initialization comment above.